tools/ci_build/github/windows/extract_nuget_files_gpu.ps1

```diff
@@ -29,14 +29,14 @@ Foreach-Object {
 New-Item -Path $Env:BUILD_BINARIESDIRECTORY\RelWithDebInfo\_deps\protobuf-build\RelWithDebInfo -ItemType directory
 
 Copy-Item -Path $Env:BUILD_BINARIESDIRECTORY\RelWithDebInfo\RelWithDebInfo\nuget-artifacts\onnxruntime-win-x64-cuda-*\lib\* -Destination $Env:BUILD_BINARIESDIRECTORY\RelWithDebInfo\RelWithDebInfo
-Copy-Item -Path $Env:BUILD_BINARIESDIRECTORY\extra-artifact\protoc.exe $Env:BUILD_BINARIESDIRECTORY\RelWithDebInfo\_deps\protobuf-build\RelWithDebInfo
+dotnet tool install --global protoc --version 3.6.1
```

Comment:
Should not be 3.6.1. Should be the one we use in cmake/deps.txt.

Reply:
The package you choose doesn't seem to be published by Google.  And it is out of maintain.

Reply:
I think you should use google.protobuf.tools. I checked, the DLLs are signed by Google.